scraper.rb

```diff
@@ -20,6 +21,11 @@ def datefrom(date)
   Date.parse(date)
 end
 
+def formatdate(date)
+  return if date.to_s.strip.empty?
+  parsed_date = datefrom(date).to_s
```

Comment:
There doesn't seem to be much value in assigning to a `parsed_date` variable here, rather than simply returning the value